pkg/volume/rbd/rbd_util.go

```diff
@@ -37,19 +38,51 @@ import (
 	"k8s.io/kubernetes/pkg/volume"
 )
 
+// search /sys/bus for rbd device that matches given pool and image
+func getDevFromImageAndPool(pool, image string) string {
+	// /sys/bus/rbd/devices/X/name and /sys/bus/rbd/devices/X/pool
+	sys_path := "/sys/bus/rbd/devices"
+	if dirs, err := ioutil.ReadDir(sys_path); err == nil {
+		for _, f := range dirs {
+			name := f.Name()
```

Comment:
Yeah you can use `|` to separate multiple regexps (e.g. `a.*b | b.*a`)

See:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/custom_column_printer.go#L41

for an example